provider-ci/internal/pkg/templates/bridged-provider/Makefile

```
@@ -440,5 +448,12 @@ provider_dist-windows-amd64: bin/$(PROVIDER)-v$(VERSION_GENERIC)-windows-amd64.t
 provider_dist: provider_dist-linux-amd64 provider_dist-linux-arm64 provider_dist-darwin-amd64 provider_dist-darwin-arm64 provider_dist-windows-amd64
 .PHONY: provider_dist-linux-amd64 provider_dist-linux-arm64 provider_dist-darwin-amd64 provider_dist-darwin-arm64 provider_dist-windows-amd64 provider_dist
 
+#{{- if .Config.RenovateCmd }}#
+renovate_cmd = #{{ .Config.RenovateCmd }}#
+renovate:
+	$(call renovate_cmd)
+.PHONY: renovate
+#{{- end }}#
```

As discussed, I think this is a backwards way of configuring renovate - we should just be building an existing makefile target via the per-repo renovate config instead.

The renovate target is fine as a temporary workaround until we figure out why renovate won't let us call `make generate`.

One suggestion here for simplifying would be to not make it an arbirary command but make it `.Config.RenovateTarget` instead so it's not arbitary code, but just an alias to an existing target.

This would also simplify the readability of the code as it's just creating an alias to an existing target:

```suggestion
#{{- if .Config.RenovateTarget }}#
renovate: #{{ .Config.RenovateTarget }}#
#{{- end }}#
```

Having the configuration as an existing target will also make it trivial in the future to remove this target and instead generate the renovate config with the call to `make #{{ .Config.RenovateTarget }}#` directly.

---

I like `make renovate` target TBH. This makes the interface between sub-systems more about the intent and less about the implementation which feels good.

---

The suggested change does make https://github.com/pulumi/pulumi-awsx/pull/1485/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R381 more difficult - we're not just generating SDKs, renovate might need to do something else specific to the repo. I think I will leave the Cmd hook. The Target hook seems to invite backdoors such as:

```
Config.RenovateTarget = "generate_sdks\n\techo STUFF\n\npost_renovate: renovate"
```

---

> The renovate target is fine as a temporary workaround until we figure out why renovate won't let us call make generate.

LMK if you still need help getting to the bottom of this. IIRC this is basically a pre-run step that we need https://github.com/renovatebot/renovate/discussions/35466 for.

---

Yeah overall we're stuck on finishing renovate rollout. It's not finished. Could use help yeah, it's not been prioritized but we can work on that - it certainly could help keeping things fresh if it just worked.